libs/ledger-live-common/package.json

```diff
@@ -258,7 +258,7 @@
     "@ledgerhq/types-cryptoassets": "workspace:^",
     "@ledgerhq/types-devices": "workspace:^",
     "@ledgerhq/types-live": "workspace:^",
-    "@solana/web3.js": "1.77.3",
+    "@solana/web3.js": "1.95.2",
```

Comment:
Not sure we want to update unrelated libs in the same PR that just touches typescript and build stuff